frontend/util.lua

```diff
@@ -597,4 +597,9 @@ function util.shell_escape(args)
     return table.concat(escaped_args, " ")
 end
 
+table.clear = function(t)
+    local c = #t
+    for i = 0, c do t[i] = nil end
```

Comment:
`for i in next, t do t[i]  = nil end`?

Reply:
`for k,v in pairs(t) do t[k] = nil end`?

Reply:
I was talking about legibility since it doesn't really feel like Lua style,[1] but if we're concerned about performance we should completely ignore Lua benchmarks and profile LuaJIT.

[1] Hm, I hope I'm not sounding too much like one of those people who's always saying something or other isn't "Pythonesque".